src/odinapi/views/read_apriori.py

```diff
@@ -30,36 +34,38 @@ def get_data(key):
     }
 
 
+def get_interpolation_weights(
```

Comment:
Is it possible to make this leapyear safe without addind much complexity?

Reply:
no. the endpoint does not know what the current year is.
As it is know the endpoint returns identical data for day 365 and 366.
I do not think this is a big issue.

Reply:
you are probably right.